docs/content/limitador-persistence.md

```diff
@@ -0,0 +1,191 @@
+
+For detailed, official instructions, refer to the Red Hat documentation:
+
+- [Red Hat Connectivity Link - Configure Redis](https://docs.redhat.com/en/documentation/red_hat_connectivity_link/1.1/html/installing_connectivity_link_on_openshift/configure-redis_connectivity-link)
+
+---
+
+## Local Validation Script (Basic Redis)
+
+The following script deploys a basic, non-production Redis instance to your current OpenShift project. This is intended for local development and validation only.
+
+Save this script as `setup-redis.sh` and make it executable (`chmod +x setup-redis.sh`). The script is located at [`deployment/scripts/setup-redis.sh`](../../deployment/scripts/setup-redis.sh).
+
+```bash
+#!/bin/bash
+#
+# This script deploys a basic Redis instance (Deployment + Service) 
+# in the current OpenShift project for testing Limitador persistence.
+#
+
+# Get the current project name
+PROJECT=$(oc project -q)
+
+if [ $? -ne 0 ]; then
+  echo "Error: Not logged into an OpenShift cluster or no project selected."
+  echo "Please log in and select a project: oc login"
+  exit 1
+fi
+
+echo "Deploying Redis Deployment and Service to project '$PROJECT'..."
+
+# Create the Redis Deployment
+oc apply -f - <<EOF
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: redis
+  labels:
+    app: redis
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: redis
+  template:
+    metadata:
+      labels:
+        app: redis
+    spec:
+      containers:
+      - name: redis
+        image: redis:7-alpine
+        ports:
+        - containerPort: 6379
+EOF
+
+# Create the Redis Service
+oc apply -f - <<EOF
+apiVersion: v1
+kind: Service
+metadata:
+  name: redis-service
+  labels:
+    app: redis
+spec:
+  selector:
+    app: redis
+  ports:
+  - protocol: TCP
+    port: 6379
+    targetPort: 6379
+EOF
+
+echo "Waiting for Redis to be ready..."
+oc wait --for=condition=Available deployment/redis --timeout=120s
+
+if [ $? -eq 0 ]; then
+  echo ""
+  echo "✅ Redis deployment successful."
+  echo ""
+  echo "Use this URL in your Limitador CR 'spec.storage.redis.config.url':"
+  echo "redis://redis-service.$PROJECT.svc:6379"
+else
+  echo "❌ Redis deployment failed or timed out."
+fi
+```
+
+---
+
+## How to Validate Persistence
+
+1. **Run the script**: `./deployment/scripts/setup-redis.sh`
+
+2. **Copy the output URL** (e.g., `redis://redis-service.my-project.svc:6379`).
+
+3. **Edit your Limitador CR** and add the storage block pointing to that URL:
+
+   ```bash
+   oc edit limitador <your-limitador-instance-name>
+   ```
+
+4. **Send traffic** against a rate-limited route until you have a non-zero hit count.
+
+   You can verify metrics in Prometheus:
+
+   ```bash
+   # Port-forward to Prometheus
+   oc port-forward -n openshift-monitoring svc/prometheus-k8s 9090:9091
```

Comment:
I wonder if it would make sense to stick to vanilla k8s (and that includes snippets) whenever possible?